src/serializers/graphcontext_type.c

```diff
@@ -35,7 +35,7 @@ static void *_GraphContextType_RdbLoad(RedisModuleIO *rdb, int encver) {
 			   REDISGRAPH_MODULE_VERSION, encver);
 		return NULL;
 		// Previous version.
-	} else if(encver >= GRAPHCONTEXT_TYPE_DECODE_MIN_V && encver < GRAPH_ENCODING_VERSION_LATEST) {
```

Comment:
What's the issue here? why aren't we validating the minimum version?

Reply:
In line 33 we check if(encver < GRAPHCONTEXT_TYPE_DECODE_MIN_V).
So if we reached line 38, the condition of line 33 must be false. Therefore checking the if encver >= GRAPHCONTEXT_TYPE_DECODE_MIN_V in line 38 is redundant.